src/lftp_ssl.cc

```diff
@@ -340,10 +340,23 @@ void lftp_ssl_gnutls::load_keys()
    }
    gnutls_credentials_set(session, GNUTLS_CRD_CERTIFICATE, cred);
 }
-void lftp_ssl_gnutls::shutdown()
+int lftp_ssl_gnutls::shutdown()
 {
+   int res;
+
    if(handshake_done)
-      gnutls_bye(session,GNUTLS_SHUT_RDWR);  // FIXME - E_AGAIN
+      res=gnutls_bye(session,GNUTLS_SHUT_RDWR);
```

Comment:
move declaration of res here, it's better to declare in a more local scope where it is used

Reply:
Do you mean:
```suggestion
      int res=gnutls_bye(session,GNUTLS_SHUT_RDWR);
```
And remove ```int res;``` from the top of ```lftp_ssl_gnutls::shutdown()```?

Reply:
Yes, that's right